supla-mqtt-client/src/json/jsoncons/basic_json.hpp

```diff
@@ -0,0 +1,5268 @@
+// Copyright 2013 Daniel Parker
```

Comment:
Maybe it is better to move supla-server / src / json to supla-common / json and use one library?
Please check whether the licenses used are compatible with the GPL. Boost is compatible
https://www.wikidata.org/wiki/Q2353141

Reply:
MQTT client library and yaml library uses MIT licence. 
Json uses Boost license. Json library is used mostly because of jsonpointer nad jsonpath implementation witch is not in supla-common/json library